lib/checkSystem.js

```diff
@@ -15,7 +14,7 @@ let check = function(pathToPackage) {
 
     const checkerResult = {
         status: 0,
-        message: '',
+        message: {},
```

Comment:
Why did this change to an object?

Reply:
@mohlsen Because in **bin/check-engine.js** you are reading _result.message.type_ and _result.message.text_ for coloring console output. So, this is typing problem.
Message cannot be an empty string by code flow in **bin/check-engine.js**